### PR TITLE
Add Sheragi Rebirth.yaml

### PR DESCRIPTION
**Pull request by Petersupes**
New plugin for Endless Sky 0.9.16.1

### CODE REVIEW

File: manifests/Sheragi Rebirth.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Sheragi Rebirth
```
Comment (quyykk) [inline manifests/Sheragi Rebirth.yaml:1]:
```suggestion
name: Sheragi Rebirth
```

Comment (Petersupes) [reply manifests/Sheragi Rebirth.yaml:1]: Hi, I'm new to using github, so any help you provide would be appreciated. The suggested change you've made doesn't appear to be any different, so if you could please elaborate on it, thank you.

Comment (quyykk) [reply manifests/Sheragi Rebirth.yaml:1]: Oh yeah GitHub seems to have messed that up. The line ending at the end is wrong.

Comment (Petersupes) [reply manifests/Sheragi Rebirth.yaml:1]: wrong in what way, and if you wouldn't mind helping me fix it.

**Review by quyykk — APPROVED** [review]
There you go, I fixed it for you

**Pull request merged**